### PR TITLE
[ADD] account,stock,account_intrastat: add Intrastat report

### PR DESCRIPTION
**Pull request by smetl**
-task: https://www.odoo.com/web#id=35850&view_type=form&model=project.task&action=333&active_id=967&menu_id=4720

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock_intrastat/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+A module that add the stock management in intrastat reports.
+============================================================
+
+This module gives the details of the goods traded between the countries of
+European Union.""",
+    'depends': ['stock_account', 'account_intrastat', 'stock_dropshipping'],
+    'data': [
+        'views/stock_warehouse_view.xml',
+    ],
+    'auto_install': True,
```
Comment (qdp-odoo) [inline addons/stock_intrastat/__manifest__.py]: can't be auto installed with dropshipping module in the dependencies

File: addons/account_intrastat/data/transport_data.xml [hunk 2]
```diff
@@ -1,35 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
   <data noupdate="1">
-    <record id="l10n_be_intrastat.intrastat_trmode_1" model="l10n_be_intrastat.transport_mode">
+    <record id="account_intrastat_transport_1" model="account.intrastat.transport">
       <field name="code">1</field>
```
Comment (qdp-odoo) [inline addons/account_intrastat/data/transport_data.xml]: are you sure it's not only valid in Belgium?

File: addons/account_intrastat/data/transaction_data.xml [hunk 3]
```diff
@@ -1,39 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
   <data noupdate="1">
-    <record id="l10n_be_intrastat.intrastat_transaction_1" model="l10n_be_intrastat.transaction">
+    <record id="account_intrastat_transaction_1" model="account.intrastat.transaction">
       <field name="code">1</field>
```
Comment (qdp-odoo) [inline addons/account_intrastat/data/transaction_data.xml]: are you sure it's not only valid in Belgium?

File: addons/l10n_be_intrastat/data/l10n_be_intrastat_chart_data.xml [hunk 4]
```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
   <data noupdate="1">
-    <record id="l10n_be_intrastat.intrastat_region_1" model="l10n_be_intrastat.region">
+    <record id="l10n_be_intrastat.intrastat_region_1" model="account.intrastat.region">
       <field name="code">1</field>
```
Comment (qdp-odoo) [inline addons/l10n_be_intrastat/data/l10n_be_intrastat_chart_data.xml]: well, that's IS for sure only valid for Belgium

File: addons/stock_intrastat/__manifest__.py (hunk 1 above)
Comment (smetl) [reply addons/stock_intrastat/__manifest__.py]: I can remove the dependency to the dropshipping module (only needed for `self.env.ref('stock_dropshipping.picking_type_dropship')`) or remove the `auto_install`.

File: addons/account_intrastat/data/transport_data.xml (hunk 2 above)
Comment (smetl) [reply addons/account_intrastat/data/transport_data.xml]: I found the same codes for Polish:
https://stat.gov.pl/en/intrastat/mode-of-transport-codes/

Germany (page 9):
https://www-idev.destatis.de/idev/doc/intra_en/doc/svz_datei_intra_en.pdf

File: addons/account_intrastat/data/transaction_data.xml (hunk 3 above)
Comment (smetl) [reply addons/account_intrastat/data/transaction_data.xml]: Same as transport codes, I found exactly the same codes in another countries

File: addons/l10n_be_intrastat/data/l10n_be_intrastat_chart_data.xml (hunk 4 above)
Comment (smetl) [reply addons/l10n_be_intrastat/data/l10n_be_intrastat_chart_data.xml]: Yeah! That's the reason this is on the `l10n_be_intrastat` module :)

**Comment by qdp-odoo** [comment]
merged into master, for saas)11.5, at revisions
https://github.com/odoo/odoo/commit/c75f3a007c89ec16d7cca4d9f80659eec9e4e772
https://github.com/odoo/enterprise/commit/16c988f519e60f626b52882f4ee32692ee62a849

**Pull request closed without merging**